opensrp-reveal/src/main/assets/json.form/bednet_distribution.json

```diff
@@ -0,0 +1,554 @@
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "hint": "Number of bed spaces",
+        "edit_type": "number",
+        "v_required": {
+          "value": true,
+          "err": "The number of bed spaces is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "calcNumNetsNeeded",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "hidden",
+        "calculation": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "bednet-distribution-calculation-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "displayRecommendedNumNets",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "toaster_notes",
+        "text": "{calcNumNetsNeeded} nets are needed",
+        "text_color": "#000000",
+        "toaster_type": "info",
+        "calculation": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "bednet-distribution-calculation-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "existingNets",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "hint": "Total number of existing Nets",
+        "edit_type": "number",
+        "value": "0",
+        "v_required": {
+          "value": true,
+          "err": "The total number of existing nets is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "existingLLINs",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "hint": "How many of these nets are LLINs or HLINs?",
+        "edit_type": "number",
+        "value": "0",
+        "v_required": {
+          "value": true,
+          "err": "The total number of existing LLINs is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        },
+        "v_cumulative_total": {
+          "value":"existingNets",
+          "related_fields": [
+            "existingITNs"
+          ],
+          "err": "The total number of existing nets should be equal to the sum of LLINs and ITNs"
+        }
+      },
+      {
+        "key": "existingLLINsLessThan1yr",
```

Comment:
@githengi or @ellykits , Akros introduced a complex set of cumulative totals. I'm getting the same issue with these cumulative totals that I got the last time I tried to create them. The logic is that the sum of all of the following fields should equal the total number of existing LLINS distributed (existingLLINs)
existingLLINS = existingLLINSLessThan1yr + existingLLINs1yrTo2yr + existingLLINs2yrTo3Yr + existingLLINsGreaterThan3yr

Can you review this in the app and see what's wrong? This is the final thing required.

Reply:
@craigappl Native form is case sensitive. The problem was the case of the related fields so the field was not being found.

it was `existingLLINs2yrTo3Yr` instead of `existingLLINs2yrTo3yr`